internal/kafka/internal/services/util.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 const (
-	truncatedNameLen = 10
+	TruncatedNameLen = 10
```

Comment:
This name was already quite generic: what is the purpose of `truncatedNameLen`? what does it define?)
Now that the constant is becoming public I think at least documenting it would be a good idea